public/lib/agg_response.js

```diff
@@ -2,48 +2,54 @@
 // _array_to_linked_list should be imported from the removed directory 'ui/agg_response/hierarchical/_array_to_linked_list'
 const { arrayToLinkedList } = require('./_array_to_linked_list');
 const { aggregate } = require('./agg_response_helper');
+require('ui/notify');
 
-module.exports = function sankeyProvider(Private, Notifier) {
-  let notify = new Notifier({
+module.exports = function sankeyProvider(Private, createNotifier) {
```

Comment:
Looks like you added the change from the other branch into this commit. This is kinda strange. Please keep different changes in different pull requests (or at least different commits).